### DISCUSSION TITLE
Adding effects to wallpaper

### DISCUSSION THREAD

**Discussion by Kozer11** [rocksdanister/lively#1719] [Q&A]
I have wallpaper (image) using lively wallpaper. Can I somehow make it video, or just add effects (like rain or storm)? Here are lively settings (pic), website ive tried to use to add effects (https://www.rocksdanister.com/lively/) [now when I click "custom" here and try to add my wallpaper to see how it'd look, nothing happens, i dont know why]
![obraz_2023-05-19_001123537](https://github.com/rocksdanister/lively/assets/134015807/c3a215e0-e8a6-481b-b7d4-e3f3aa972b28)

**Comment by rocksdanister** [comment]
I don't get you, in this video I am adding a custom picture to "Rain":

https://github.com/rocksdanister/lively/assets/17554161/9d091a0d-95d2-4f85-aa58-72908114b13f

For adding video files it need to be converted to .webm file:
https://github.com/rocksdanister/lively/wiki/Web-Guide-VI-:-Webpage-Video-Support#converting-existing-videos-to-webm

Is this what you need? if not then try recording a video showing what you did in https://www.rocksdanister.com/lively/
